scipy/signal/_peak_finding.py

```diff
@@ -233,6 +236,733 @@ def argrelextrema(data, comparator, axis=0, order=1, mode='clip'):
+        supplied, as the maximum required prominence.
+    width : number or ndarray or sequence, optional
+        Required width of peaks. Either a number, ``None``, an array
+        matching `x` or a 2-element sequence of the former. The first
+        element is always interpreted as the  minimum and the second, if
+        supplied, as the maximum required prominence.
+    wlen : number, optional
+        Used for calculation of the peaks prominences thus it is only used if
+        one of the arguments `prominence` or `width` is given. See argument
+        `wlen` in `peak_prominences` for a full description of its effects.
+    rel_height : float, optional
+        Used for calculation of the peaks width thus it is only used if `width`
+        is given. See argument  `rel_height` in `peak_widths` for a full
+        description of its effects.
+
+    Returns
+    -------
+    peaks : ndarray
+        Indices of peaks in `x` that satisfy all requirements.
+    properties : dict
+        A dictionary containing properties of the returned peaks which were
+        calculated as intermediate results during filtering:
+
+        * If `height` is given, the value of `x` at each peak is returned
+          with the key 'peak_heights'.
+        * If `threshold` is given, the keys 'left_thresholds' and
+          'right_thresholds' contain a peaks vertical distance to its
+          neighbouring samples.
+        * If `prominence` is given, the keys 'prominences', 'left_bases' and
+          'right_bases' are accessible. See `peak_prominences` for a description
+          of their content.
+        * If `width` is given, the keys 'wheights', 'left_ips' and 'right_ips'
+          are accessible. See `peak_widths` for a description of their content.
+
+    See Also
+    --------
+    find_peaks_cwt
+        Find peaks using the wavelet transformation.
+    peak_prominences
+        Directly calculate the prominence of peaks.
+    peak_widths
+        Directly calculate the width of peaks.
+
+    Notes
+    -----
+    Because this function searches for local maxima by direct sample comparison,
+    the determined peak locations can be off for noisy signals if the noise
+    changes the position of a local maximum. In those cases, smoothing the signal
+    may be more appropriate before searching for peaks or using other peak
+    finding and fitting methods (like `find_peaks_cwt`).
+
+    Some additional comments on filtering with requirements:
+
+    * Almost all filter options (excluding filtering by distance) allow the
+      specification of half-open or closed intervals, e.g ``1`` or ``(1, None)``
+      define the half-open interval :math:`[3, \\infty]` while ``(None, 1)``
+      defines the interval :math:`[-\\infty, 3]`.
+    * The border is always included in the interval used to select valid peaks.
+    * The order of input arguments for the different filters mirrors the actual
```

Comment:
The order of the input arguments *given in the function definition above* for the ....

Otherwise someone might think that doing `height=a, threshold=b` will do something different from `threshold=b, height=a`